spec/controllers/comments_controller_spec.rb

```diff
@@ -15,19 +15,23 @@
       end
 
       it "sends a comment email to approvers and observers" do
+        ENV["DISABLE_EMAIL"] = nil
```

Comment:
Not sure if it'll totally get us there — https://relishapp.com/rspec/rspec-core/v/3-4/docs/metadata/user-defined-metadata